### PR TITLE
update gm-data to 1.1.5

### PR DESCRIPTION
**Pull request by tilleryd**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Updates gm-data to 1.1.5.

2. What **changes to custom.yaml** are required?

none

3. Have you documented any additional setup steps or configurations options?

n/a

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

n/a

### CODE REVIEW

**Review by zoemccormick — COMMENTED** [review]
looks good! You will want to update the following too to reflect in the dashboard:

1. https://github.com/greymatter-io/helm-charts/blob/a717887d49b91c69bdc0a70ec4393a6a7001259b/fabric/values.yaml#L366
2. https://github.com/greymatter-io/helm-charts/blob/a717887d49b91c69bdc0a70ec4393a6a7001259b/fabric/control-api/values.yaml#L310
3. https://github.com/greymatter-io/helm-charts/blob/a717887d49b91c69bdc0a70ec4393a6a7001259b/sense/values.yaml#L512
4. https://github.com/greymatter-io/helm-charts/blob/a717887d49b91c69bdc0a70ec4393a6a7001259b/sense/catalog/values.yaml#L278

**Review by zoemccormick — APPROVED** [review]
lgtm

**Comment by tilleryd** [comment]
I've gotten all GitHub Action checks to pass, which was really helpful, thanks @chrisbsmith. @zoemccormick can you please re-review?

File: data/gm-data/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 1.0.0
+appVersion: 1.1.5
```
Comment (chrisbsmith) [inline data/gm-data/Chart.yaml:2]: good catch

**Review by chrisbsmith — APPROVED** [review]
LGTM

**Pull request merged**